pyproject.toml

```diff
@@ -0,0 +1,9 @@
+[tool.black]
+
+include = '''
+.*\.py$
+'''
+
+force-exclude = '''
+^(?!\/players\/).*
+'''
```

Comment:
This is to disable automatic formatting of global code for users with automatic IDE formatting using [black](https://black.readthedocs.io/en/stable/index.html)